scripts/valhalla_build_config

```diff
@@ -236,7 +236,8 @@ config = {
       'max_contours': 4,
       'max_time': 120,
       'max_distance': 25000.0,
-      'max_locations': 1
+      'max_locations': 1,
+      'max_distance_contours': 120
```

Comment:
hmpf.. in the code I actually named it `max_kilometers` not `max_distance_contours`.. any preference? or smth else entirely?

Reply:
im not sure we even need a new config parameter here? max_distance already specifies the largest distance contour value. max_contours still specifies the max number of contours regardless of type (time or distance you only get 4 total by default). maybe you want to change the default of the secone one but even that i think is fine to stay 4 by default

Reply:
I would recommend a new one! max_distance specifies the largest distance between input locations (I guess to sorta limit the size of the data grid). one could use simply max_time, but that's a little confusing, and for slower costing models (bikes, ped) max_time and max_distance are quite a different concept. I think 4 input contours are fair still

Reply:
ah right sorry for the confusion. its just an unfortunate naming clash. maybe we could rename `max_time` to `max_time_contour` and then add a `max_distance_contour`?

Reply:
yeah, agree, unfortunate naming.. won't that be a breaking change for all existing config files then? right now the `max_time` setting is required in loki_worker. we'd need to make it optional and provide a default in loki_worker if we'd want to rename right?

Reply:
or duplicate the max_time with a new name in the config. both a little ugly IMO. but maybe as ugly as the current naming;)

Reply:
yes it would be but we break configs all the time by adding new required ones. there currently no real way around it so i think its the cost of doing business. the good news is that running code with a malformed config immediately fails and you get an error message why

Reply:
done